python/feathub/feathub_client.py

```diff
@@ -28,6 +28,57 @@
 from feathub.table.table_descriptor import TableDescriptor
 
 
+class ProcessorJobGroup:
```

Comment:
It might be better to change the signature of the methods into follows:
```python
class ProcessorJobGroup:
    def add_job(
        self,
        feature_descriptor: Union[str, TableDescriptor],
        sink: FeatureTable,
        ttl: Optional[timedelta] = None,
        start_datetime: Optional[datetime] = None,
        end_datetime: Optional[datetime] = None,
        allow_overwrite: bool = False,
    ) -> None:
       ...

    def materialize(self) -> ProcessorJob:
        pass

```